LayoutTests/http/tests/inspector/dom/site-isolated-node.html

```diff
@@ -0,0 +1,43 @@
+<!-- webkit-test-runner [ SiteIsolationEnabled=true ] -->
+<!doctype html>
+<html>
+<head>
+<script type="text/javascript" src="../resources/inspector-test.js"></script>
+<script>
+function test()
+{
+    let suite = InspectorTest.createAsyncSuite("CommandLineAPI.$0.cross-frame");
+
+    let mainFrame = WI.networkManager.mainFrame;
```

Comment:
id recommend putting all this in a test case instead of at the top-level
```js
suite.addTestCase({
    name: "AttemptCrossFrame$0AccessFromMainFrame",
    description: "Should not be able to access $0 node in different domain subframe from the main frame.",
    async test() {
        let mainFrame = WI.networkManager.mainFrame;
        let childFrames = Array.from(mainFrame.childFrameCollection);
        InspectorTest.expectEqual(childFrames.length, 1, "Page should have a subframe.");

        let documentNode = await WI.domManager.requestDocument();
        let iframeNodeId = await documentNode.querySelector("iframe#myframe");

        InspectorTest.log("Setting $0 to subframe...");
        await DOMAgent.setInspectedNode(iframeNodeId);

        let {result, wasThrown} = await RuntimeAgent.evaluate.invoke({expression: "$0", includeCommandLineAPI: true});
        InspectorTest.assert(!wasThrown, "Should not be an exception.");
        InspectorTest.expectEqual(result.subtype, InspectorBackend.Enum.Runtime.RemoteObjectSubtype.Node, "Should be able to access subframe.");
    },
});
```